src/features/user/Widget/EmbedModal.tsx

```diff
@@ -1,10 +1,10 @@
-import { Modal, Snackbar } from '@material-ui/core';
 import React, { ReactElement } from 'react';
+import { Modal, Snackbar } from '@mui/material';
```

Comment:
As with the other modals, there are slight differences in font sizes, font styles, spacing, height, width.
![Screenshot 2022-05-05 at 5 53 50 PM](https://user-images.githubusercontent.com/44917347/166922213-25fa6308-0eed-4d62-a4d9-16718b362263.png)

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467